src/Promenade.Models/Entities/PageLayout.cs

```diff
@@ -30,5 +30,10 @@ public class PageLayout
 
         [NotMapped]
         public PageLayoutText PageLayoutText { get; set; }
+
+        public Guid PreviewId { get; set; }
+
+        [NotMapped]
+        public bool IsPreview { get; set; }
```

Comment:
I think this field can be removed, it doesn't like its value is being used anywhere.

Reply:
I think it's used in PageService.cs and LayoutPage.cshtml.